packages/design-picker/src/components/index.tsx

```diff
@@ -26,7 +26,7 @@ const makeOptionId = ( { slug }: Design ): string => `design-picker__option-name
 interface Props {
 	locale: string;
 	onSelect: ( design: Design ) => void;
-	designs: Design[];
+	designs?: Design[];
```

Comment:
This has to be optional since we're not always passing designs as prop to the component.

Reply:
That's what I though too, checks out

Reply:
Gotta love tests for defining clear answers to this!